draft-wussler-openpgp-pqc.md

```diff
@@ -1321,7 +1323,7 @@ when using a sponge-based construction like KMAC. The construction using KMAC
 ensures a standardized domain separation. In this case, the processed message
 is then the concatenation of any number of key shares.
 
-More precisely, for given capacity `c` the indifferentiability proof shows that
+More precisely, for a given capacity `c` the indifferentiability proof shows that
```

Comment:
I think the 'a' is not necessary here.

Reply:
Also to me it "sounds better" with the `a` but I see Stavros' point

Reply:
```suggestion
More precisely, for given capacity `c` the indifferentiability proof shows that
```
I don't know english grammar well enough to be sure but also I don't mind too much. Maybe we get a review from an english native speaker at some point.

Reply:
For me introducing an 'a' is fine.